### PR TITLE
Resolve issue when decoding id tokens

### PR DESCRIPTION
**Pull request by thivi**
## Fix
Used the Crypto JS library's `Base64URLDecode` method to decode the payload of the ID token. In order to support other languages, the decoded payload was also utf8 encoded.

### CODE REVIEW

File: .github/workflows/bump-version.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+          git config --global user.name "asgardio-github-bot"
+          npm run bump-minor-version
+          git push --follow-tags
+        if: github.event.inputs.Type == 'minor'
+        
+      - name: Bump Major Version
+        run: |
+          git config --global user.email "version.bump@github.action.com"
+          git config --global user.name "asgardio-github-bot"
+          npm run bump-major-version
```
Comment (brionmario) [inline .github/workflows/bump-version.yml:52]: Does actions allow this kind of syntax. So, we can get rid of the duplicated steps.
```suggestion
          npm run bump-${{ github.event.inputs.Type }}-version
```

File: packages/oidc-js/src/utils/sign-in.ts [hunk 2]
```diff
@@ -694,6 +701,20 @@ export const getUserInfo = (config: ConfigInterface | WebWorkerConfigInterface):
...
+ * This function decodes the payload of an id token and returns it.
+ *
+ * @param {string} idToken - The id token to be decoded.
+ *
+ * @return {DecodedIdTokenPayloadInterface} - The decoded payload of teh id token.
+ */
+const decodeIDToken = (idToken: string): DecodedIdTokenPayloadInterface => {
+    const words = base64URLDecode.parse(idToken.split(".")[ 1 ]);
+    const utf8String = utf8.stringify(words);
+    const payload = JSON.parse(utf8String);
```
Comment (brionmario) [inline packages/oidc-js/src/utils/sign-in.ts:714]: Would be better to add a try catch here.

**Review by brionmario — APPROVED** [review]

File: packages/oidc-js/src/utils/sign-in.ts (hunk 2 above)
Comment (thivi) [reply packages/oidc-js/src/utils/sign-in.ts]: Fixed by 3aaab96

File: .github/workflows/bump-version.yml (hunk 1 above)
Comment (thivi) [reply .github/workflows/bump-version.yml:52]: Resolved by 025c819

**Pull request merged**